src/playbacks/html5_video/html5_video.js

```diff
@@ -695,6 +730,15 @@ export default class HTML5Video extends Playback {
     }
 
     this._ready()
+
+    // Note: Example of Picture-in-Picture
+    // const pipButton = document.createElement('button', { class: 'pip-button' })
+    // document.body.appendChild(pipButton)
+    // pipButton.innerHTML = 'Picture-in-Picture'
+    // pipButton.addEventListener('click', () => {
+    //   this.togglePIP()
+    // })
```

Comment:
remove this unecessary comment

Reply:
Done in [Commit](https://github.com/clappr/clappr-core/pull/120/commits/7d8e32492c1d895c10d1b9200e151bcaf1c01e00)